work_items/trust_establishment.md

```diff
@@ -0,0 +1,62 @@
+# Trust Establishment
+
+Trust in the decentralized space is a wide problem that many have tried to solve. In this work item we aim to take a piece of the problem around _trust establishment_. Decentralized trust establishment is the means by which parties answer two key questions about one another:
+
+1. Are you who you claim to be?
+2. Are you to be trusted for what you claim to be trusted for?
+
+
+## Work Item Owners
+- Gabe Cohen (@decentralgabe)
+- ...
+
+## Outcome
+**When your work item is successful, then what will have changed in the world? What will we be able to do that wasn't possible do before or vice versa?**
+
+Parties using Decentralized Identifier-relative protocols, and the applications build on top of them, will be able to interrogate, investigate, and establish trust in an common, standards-based method.
+
+## Deliverables
+- **Does your work item have any deliverables? These include code projects,
+  standards-track specifications, reports, registries, summaries, blog posts,
+  and anything else that is instantiable and publishable in written form to a
+  broader audience.**
+
+  We aim to at least deliver a standard format for representing Trust Lists. 
+  Expanding on Trust Lists, we aim to produce an implementer's guide for creating, using, and evolving Trust Lists. This implementers guide is likely to touch on prior art such as the [Well Known DID Configuration](https://identity.foundation/.well-known/resources/did-configuration/) and other existing means for trust establishment.
+
+- **What's the expected timeline?**
+
+  We aim to publish a first draft by the end of Q3, 2022 -- June 15.
```

Comment:
Confusing with end of Q3 and june 15? End of Q3 is late september

Reply:
Might we be using the lunar calendar? Joking aside, end of Q**2** -- June 15-July 1st seems like a healthy timeline to have a first draft up

Reply:
Oops yes, Q2